.travis/install.sh

```diff
@@ -17,7 +17,8 @@ fi
 # Install wireshark data
 if [ "$TRAVIS_OS_NAME" = "linux" ] && [ "$TRAVIS_SUDO" = "true" ]
 then
-  sudo apt-get -qy install tshark
+  # The wireshark packets should be installed with tshark, but Travis sometimes ignores the dependencies
+  sudo apt-get -qy install libwireshark5 wireshark-common tshark
```

Comment:
See https://travis-ci.org/gpotter2/scapy/jobs/390697554
```
$ bash .travis/install.sh
Reading package lists...
Building dependency tree...
Reading state information...
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:
The following packages have unmet dependencies:
 tshark : Depends: libwireshark5 (>= 1.12.0~rc3) but it is not going to be installed
          Depends: wireshark-common (= 1.12.1+g01b65bf-4+deb8u11ubuntu0.14.04.1) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
```